app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss

```diff
@@ -54,8 +54,8 @@
 
 .remove-issue-fields-button {
   position: absolute;
-  right: -4px;
-  top: 25px;
+  right: 40%;
```

Comment:
Not the crosses locates in diffrent places in show less/more view.

Show less:

![image](https://user-images.githubusercontent.com/31861805/120638170-79972400-c478-11eb-9f70-da101c39c808.png)

Show more:

![image](https://user-images.githubusercontent.com/31861805/120638246-8fa4e480-c478-11eb-88f9-d24e36132cef.png)

May be better to use static indentation for the crosses.
Just add `display: inline-block` to the `issue-fields-list-item` and `right: -50px` to the `remove-issue-fields-button`.